share/csm_share/shr/shr_flux_mod.F90

```diff
@@ -267,7 +277,15 @@ real(R8)    :: psix2  ! stability function at ztref reference height
      if (mask(n) /= 0) then
     
         !--- compute some needed quantities ---
-        vmag   = max(umin, sqrt( (ubot(n)-us(n))**2 + (vbot(n)-vs(n))**2) )
+
```

Comment:
Just as an additional note: It gives the right result only because ugust is zero. It would be very easy for this to change in the future and for CESM to suddenly have a different calculation in a very critical part of the code. I have proposed a very simple approach to
separate out what acme and cesm each want to do. But maybe its not the best
one.